autoload/tagbar.vim

```diff
@@ -2061,34 +2063,33 @@ function! s:PrintHelp() abort
         silent 0put ='\" Tagbar keybindings'
         silent  put ='\"'
         silent  put ='\" --------- General ---------'
-        silent  put ='\" ' . s:get_map_str('jump') . ': Jump to tag definition'
-        silent  put ='\" ' . s:get_map_str('preview') . ': As above, but stay in'
-        silent  put ='\"    Tagbar window'
-        silent  put ='\" ' . s:get_map_str('previewwin') . ': Show tag in preview window'
-        silent  put ='\" ' . s:get_map_str('nexttag') . ': Go to next top-level tag'
-        silent  put ='\" ' . s:get_map_str('prevtag') . ': Go to previous top-level tag'
-        silent  put ='\" ' . s:get_map_str('showproto') . ': Display tag prototype'
-        silent  put ='\" ' . s:get_map_str('hidenonpublic') . ': Hide non-public tags'
+        if !empty(s:get_map_str('jump')) | silent  put ='\" ' . s:get_map_str('jump') . ': Jump to tag definition' | endif
```

Comment:
This and the following ~28 lines are 95% repeated boiler plate. Perhaps we could abstract this into a function and just iterate over a list doing the same gymnastics for each one with only 5% of the code. Not a blocker if you don't have time to refactor this, but it would sure make this a lot prettier.

Reply:
perhaps tidy up those into a func may make it looks clean, but keyword & description each of them was different, perhaps would not reduce items/letters too much, specially to me, i was trying to reduce as less modification as possible to make it less side-effect. so perhaps just let it just like that for now, or may refactor this help func again in future with others.

Reply:
The different keyword and description pairs could be in a dictionary list that was iterated over. The code logic (`if`, `put`, etc.) is all 100% identical and could be abstracted to a function. I've opened #681 to track this if anybody wants to tackle it.

Reply:
keyword &  description were there already before this pr, i did not want to change too much;
compared to author, to me, i think i'd better less modification (not refactor too much) as better,
or author (or you?) to refactor perhaps better, since you knew that should be more clear than me.
anyway, thanks merged this.